frontend/src/components/worksheets/WorksheetDialogs.js

```diff
@@ -253,6 +259,39 @@ class WorksheetDialogs extends React.Component {
                         </DialogActions>
                     </DialogContent>
                 </Dialog>
+                {/* Specify path for content block */}
+                <Dialog
+                    open={this.props.openedDialog === DIALOG_TYPES.OPEN_CREATE_CONTENT}
+                    onClose={this.props.closeDialog}
+                    aria-labelledby='create-content-block-title'
+                    aria-describedby='create-content-block-description'
+                >
+                    <DialogTitle id='create-content-block-title' style={{ color: 'red' }}>
+                        Specify the subpath inside the bundle you want to display content for
+                    </DialogTitle>
+                    <DialogContent>
+                        <TextField
+                            autoFocus
+                            onChange={this.setPathValue}
```

Comment:
I think we can inline this function (just add it directly here instead of defining it above). It's a bit confusing to have to scroll all the way up to line 29 for a one-line function that's used once.